### DISCUSSION TITLE
Help with troubleshooting the issue

### DISCUSSION THREAD

**Discussion by rua0ra1** [cyberbotics/wrestling#65] [Q&A] [answered]
Hi,
I am  getting  the following error when I am testing my controller in the server. But, I didn't face any issues while running it in my local machine. I tried to resolve this issue and I didn't get what is the actual problem. I really appreciate it if you can help me this.
![controller_error](https://user-images.githubusercontent.com/67779589/227771545-05bd9aa4-b107-4b50-bff6-4d7911328b59.png)

**Comment by omichel** [comment, marked as answer]
You should click on the ► icon on the line:
```
► Running Game in Webots: RamCharanThota version Merak
```
To display more details about the log of the game and understand what's wrong with your controller.

**Reply by omichel** [reply]
Could please give a link to the log file so that I can also inspect it?

**Reply by rua0ra1** [reply]
Thanks @omichel ,
Please find the link to the log file below
https://github.com/cyberbotics/wrestling/actions/runs/4524197844/jobs/7967854615

**Reply by omichel** [reply]
It seems there is a problem with the vulcanexus image:
```
/vulcanexus_entrypoint.sh: line 20: exec: --: invalid option
exec: usage: exec [-cl] [-a name] [command [argument ...]] [redirection ...]
```

**Reply by jsan-rt** [reply]
Hi everyone, 

Looking at the logs it seems a Docker parameter (--cpuset-cpus) is being passed as command to the Vulcanexus entrypont. I think the line:

docker run --rm --gpus all --memory=6g --network none --volume /tmp/webots-1234/ipc/participant:/tmp/webots-1234/ipc/participant participant-controller --cpuset-cpus=2-4

should read:

docker run --rm --gpus all --memory=6g --network none --volume /tmp/webots-1234/ipc/participant:/tmp/webots-1234/ipc/participant --cpuset-cpus=2-4 participant-controller

**Reply by rua0ra1** [reply]
Thank @omichel  and @jsantiago-eProsima . Sorry for my lame question.I am just confused. Is it something I can fix it from my side?

**Reply by rua0ra1** [reply]
I am feeling like I better switch to python from ros2. I am spending more time on the build and run issues rather than on actual algorithm :(

**Reply by omichel** [reply]
Thank you @jsantiago-eProsima for the quick reaction. I will fix it now.

**Reply by omichel** [reply]
Fixed in https://github.com/cyberbotics/competition-record-action/pull/28.

**Reply by omichel** [reply]
@RamCharanThota: I am sorry for this. But now, it should be OK to run your ROS 2 controller. I hope you can switch back to ROS 2 as it is very nice to have some ROS 2 participants out there.

**Reply by rua0ra1** [reply]
@omichel Thanks for the help. I really appreciate it. Even I am very interested to design the controller in ros2. But, I don't have much experience with ROS2 or webots. So, I am spending lot of time troubleshooting build and run problems. But, I will do my best to switch to ROS2 eventually
